pandas/core/groupby/groupby.py

```diff
@@ -1330,7 +1330,9 @@ def any(self, skipna: bool = True):
 
         Returns
         -------
-        bool
+        Series or DataFrame
+            DataFrame or Series of boolean values, which is True if any element is
```

Comment:
What do you think about: "..., where a value is True if any element is True within its respective group..."

Reply:
Sounds more precise than the current one. Thank you!